packages/lumx-react/src/components/link/Link.test.tsx

```diff
@@ -46,8 +46,10 @@ describe(`<${Link.displayName}>`, () => {
     // 1. Test render via snapshot (default states of component).
     describe('Snapshots and structure', () => {
         it('should render correctly', () => {
-            const { wrapper } = setup();
+            const linkRef = createRef<HTMLAnchorElement>();
+            const { wrapper } = setup({ linkRef });
             expect(wrapper).toMatchSnapshot();
+            expect(linkRef).toBeTruthy();
```

Comment:
Should we test that the link has the prop ref instead?

Reply:
Oh actually what I meant to test is the `linkRef.current` value which would indicate that the ref has been passed to the anchor and then the current value was updated.
I'm not sure which is best. Testing if the ref is passed to the anchor or testing is if the ref current value is update with the HTMLAnchorElement DOM object :thinking:

Reply:
Maybe it's equivalent in terms of what we test. 
But I guess that the responsibility of the component is to give the ref to the proper element

Reply:
I tried testing the "ref" prop as you say but it seems to be a special prop not directly accessible via the ezyme wrapper. 
So I decided to test that the linkRef.current was correctly filled with the link DOM object reference.
Let me know what you think of this!